lib/elftools/elf_file.rb

```diff
@@ -285,7 +285,7 @@ def segment_at(n)
     # This method should work no matter ELF is a PIE or not.
     # This method refers from (actually equals to) binutils/readelf.c#offset_from_vma.
     # @param [Integer] vma The virtual address to be queried.
-    # @return [Integer] Related file offset.
+    # @return [Integer] Related file offset. nil if non-valid vma or size overlaps segments.
```

Comment:
Change return type to `Integer?` (question mark means it may return nil)

Reply:
Changed, also fixed rubocop line length warning